test/test_regex.py

```diff
@@ -83,9 +110,41 @@ def test_gdb_stripped_nonpic(self):
         self.assertEqual(call_match[1], '555555556188')
         self.assertEqual(call_match[2], '0x555555555542')
 
+    @unittest.expectedFailure
+    def test_gdb_indirect_call(self):
+        line = '0x0000000000001013 <+19>:	callq  *(%rsi)'
+        call_match = self.strip_regex.match(line)
+
+        self.assertIsNot(call_match, None)
+        self.assertEqual(call_match[1], '19')
+        self.assertEqual(call_match[2], '')
+
+        line = '0x0000000000001013 <+19>:	callq  *0x8(%rbx)'
+        call_match = self.strip_regex.match(line)
+
+        self.assertIsNot(call_match, None)
+        self.assertEqual(call_match[1], '19')
+        self.assertEqual(call_match[2], '')
+
+    @unittest.expectedFailure
+    def test_objdump_indirect_call(self):
+        line = '1013:	ff 16                	callq  *(%rsi)'
```

Comment:
Same here. How these should be handled in CFG when we get to that point. Any ideas?